frontend/components/Header/index.tsx

```diff
@@ -57,7 +57,7 @@ function Header() {
       spacing={{ xs: 1, md: 2 }}
       sx={{
         position: 'fixed',
-        zIndex: 2,
+        zIndex: 100,
```

Comment:
In general I feel like we might be overusing zIndex? Not looking at this one in particular, but overall we've ran into a few issues stemming from excessive zIndex-ing.

Reply:
The point was to be the highest on the entire site, no matter what other components are doing. This allows devs to be a little excessive with zIndex when developing components, without having to worry about screwing up menu and header overlap.